bigquery/unit_tests/test__helpers.py

```diff
@@ -491,6 +491,363 @@ def test_instance_setter_w_bad_udfs(self):
         self.assertEqual(instance.udf_resources, [])
 
 
+class Test_AbstractQueryParameter(unittest.TestCase):
+
+    @staticmethod
+    def _get_target_class():
+        from google.cloud.bigquery._helpers import AbstractQueryParameter
```

Comment:
Throughout the tests, can we add a space between the imports and the following code?